style.css

```diff
@@ -0,0 +1,100 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+@media (max-width: 768px) {
+  main {
+    background-image: url(assets/header-illsutration-mobile.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  animation-duration: 0.5s;
+  animation-name: obj-scale;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  animation-duration: 0.5s;
+  animation-name: obj-scale;
+}
+
+.primary {
+  text-align: center;
+  max-width: 327px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 45px;
```

Comment:
- [ ] Kindly use the correct font size 40px to be similar to the Figma template.